### DISCUSSION TITLE
Got error while register user using FabricCAServices

### DISCUSSION THREAD

**Discussion by prakash204** [hyperledger/fabric-gateway#135] [Q&A]
Failed to register the username NpPharma for organization Manufacturer with::fabric-ca request register failed with errors [[ { code: 71, message: 'Authorization failure' } ]]

NpPharma is userID
Manufacturer is organization name

Please let me know

**Comment by bestbeforetoday** [comment]
This discussion area is for the new Fabric Gateway client API currently under development, not for the existing client SDKs.
